sdk/storage/azure-storage-blobs/src/blob_client.cpp

```diff
@@ -61,8 +61,7 @@ namespace Azure { namespace Storage { namespace Blobs {
     }
     policies.emplace_back(std::make_unique<StoragePerRetryPolicy>());
     policies.emplace_back(std::make_unique<SharedKeyPolicy>(credential));
-    policies.emplace_back(std::make_unique<Azure::Core::Http::TransportPolicy>(
-        std::make_shared<Azure::Core::Http::CurlTransport>()));
+    policies.emplace_back(std::make_unique<Azure::Core::Http::TransportPolicy>());
```

Comment:
Doesn't this need the transport options passed to it?

Reply:
Victor mentioned [above](https://github.com/Azure/azure-sdk-for-cpp/pull/889#discussion_r518913202), there is already a PR for that: https://github.com/Azure/azure-sdk-for-cpp/pull/864